rholang/src/main/bnfc/rholang_mercury.cf

```diff
@@ -41,7 +41,7 @@ PNeq.         Proc6  ::= Proc6 "!=" Proc7 ;
 PAnd.         Proc5  ::= Proc5 "and" Proc6 ;
 POr.          Proc4  ::= Proc4 "or" Proc5 ;
 PSend.        Proc3  ::= Name Send "(" [Proc] ")" ;
-PContr.       Proc2 ::= "contract" Name "(" [Name] ")" "=" "{" Proc "}" ;
+PContr.       Proc2 ::= "contract" Name "(" [Name] Remainder")" "=" "{" Proc "}" ;
```

Comment:
Would it make more sense to have an optional Remainder instead of letting the Remainder be ""?